omnibus/.kitchen.yml

```diff
@@ -25,16 +25,52 @@ provisioner:
 platforms:
   - name: centos-6.7
     run_list: yum-epel::default
+    attributes:
```

Comment:
if these attributes are the same accross platforms, we should move them to the suite